partials/tabs/projects.html

```diff
@@ -6,10 +6,7 @@
         <input ng-model="searchText" placeholder="Search for a project" id="search" type="text" class="validate">
       </div>
       <div class="input-field col-md-2">
```

Comment:
Add a center-block here as well 👍 can you also post a screenshot of this page after changes